layers.qmd

```diff
@@ -176,7 +172,7 @@ You'll need to pick a value that makes sense for that aesthetic:
 #|   point-up blue, 25 - filled triangle point down blue.
 
 shapes <- tibble(
```

Comment:
That ordered was carefully picked to put similar shapes next to each other. I'd prefer to leave it as is, and call out the non-ascending order in the legend.

Reply:
I figured as much. I think both approaches make sense, but I reverted to the original.